app/presenters/volunteer_stats_presenter.rb

```diff
@@ -0,0 +1,41 @@
+class VolunteerStatsPresenter
+  def initialize(volunteer, time = Time.zone.now)
+    @volunteer = volunteer
+    @time = time
```

Comment:
Is there a better name for this time - `@now`?  `@current_time`?

Non-blocking